card/mod/05_standard/set/self/signin.rb

```diff
@@ -98,11 +106,13 @@ def default_core_args args={}
   abort :success
 end
 
-event :send_reset_password_token, before: :signin, on: :update, when: proc{ |c| Env.params[:reset_password] } do
-  email = subcards["+#{Card[:email].name}"]
-  email &&= email['content']
+event :send_reset_password_token,
+      before: :signin, on: :update,
+      when: proc { |c| Env.params[:reset_password] } do
```

Comment:
Unused block argument - `c`. You can omit the argument if you don't care about it.